client/src/views/v2/DownloadFilesView.js

```diff
@@ -11,38 +11,19 @@ import { useFiles, useLatestFile } from 'api/v2/files';
  * @returns {React.Component} Component
  */
 export default function DownloadFilesView() {
-  const { files, deleteFile } = useFiles();
+  const { loading, files, deleteFile } = useFiles();
   const latestFileURL = useLatestFile();
   const [deletingFile, setDeletingFile] = useState(null);
 
   const hideConfirmDelete = useCallback(() => {
```

Comment:
I'm not sure if this needs a `useCallback` since it doesn't seem to be used in a hook, maybe @angustrau can correct me on this?

Reply:
This does need useCallback as it is passed to a component. If it isn't wrapped, it would cause the child component to rerender on every render